test/handler.test.js

```diff
@@ -628,6 +628,30 @@ describe('strong-error-handler', function() {
         .expect('Content-Type', /^application\/json/, done);
     });
 
+    it('dishonors accepted content-type if mandated by contentType option',
+      function(done) {
+        givenErrorHandlerForError(new Error('Some error'), {
+          contentType: 'application/json',
+        });
+        request.get('/')
+          // `application/json` will be used because its provided first
+          .set('Accept', 'text/html')
+          .expect('Content-Type', /^application\/json/, done);
+      }
+    );
+
+    it('dishonors default content-type if mandated by contentType option',
+      function(done) {
+        givenErrorHandlerForError(new Error('Some error'), {
+          defaultType: 'application/json',
+          contentType: 'text/html',
```

Comment:
I find the option name `contentType` too vague, it does not express the intention well. How about `forceContentType`?